### ISSUE TITLE
dev-libs/glib python-any-r1_pkg_setup

### ISSUE THREAD

**Issue by c4pp4** [shiznix/unity-gentoo#213]
https://github.com/shiznix/unity-gentoo/blob/54dbe1e7c66e0216766d5e55ead00ea116e9f55f/dev-libs/glib/glib-2.58.1_p_p_p2.ebuild#L92
The line is causing `command not found` because the command is not inherited from `python-any-r1.eclass`. I'm not sure how to fix it.

**Comment by shiznix** [comment]
Oops, should be fixed in e43f5cfd4f0ad254b37f3565f0cb98d8ae01e23e

**Comment by c4pp4** [comment]
I was trying this solution but I got:
```
WARNING: One or more updates/rebuilds have been skipped due to a dependency conflict:

dev-libs/glib:2

  (dev-libs/glib-2.58.1_p_p_p2:2/58.1::unity-gentoo, ebuild scheduled for merge) conflicts with
    dev-libs/glib:2[python_targets_python2_7(-),-python_single_target_python2_7(-)] required by (unity-base/compiz-0.9.13.1_p20180930_p0_p01:0/0.9.13.1::unity-gentoo, installed)
                                                                                   
    dev-libs/glib:2[python_targets_python2_7(-),-python_single_target_jython2_7(-),-python_single_target_pypy(-),-python_single_target_pypy3(-),-python_single_target_python3_5(-),-python_single_target_python3_6(-),-python_single_target_python3_7(-),python_single_target_python2_7(+)] required by (unity-base/hud-14.10_p20170619_p0_p02-r2:0/0::unity-gentoo, installed)

```
